packages/x11/driver/xf86-video-nvidia-legacy/package.mk

```diff
@@ -3,8 +3,8 @@
 # Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="xf86-video-nvidia-legacy"
-PKG_VERSION="340.108"
-PKG_SHA256="995d44fef587ff5284497a47a95d71adbee0c13020d615e940ac928f180f5b77"
+PKG_VERSION="390.141"
```

Comment:
isn't Nvida ION not supported anymore at 390 - thats why we stuck at 340?

Reply:
👍

so it’s only the **not legacy** with the issue.

https://forum.libreelec.tv/thread/23269-nightly-builds-not-working-with-nvidia/?postID=148680#post148680

dropped legacy commit.